### PR TITLE
Update ExtUtils-MakeMaker to CPAN version 7.48

### PR DESCRIPTION
**Pull request by toddr**
[DELTA]

7.48    Sun  4 Oct 11:39:22 BST 2020

    No changes since v7.47_12

7.47_12 Wed 30 Sep 16:19:25 BST 2020

    Bug fixes:
    - Make Mkbootstrap.pm strict

    Test fixes:
    - Skip the failing dynamic vs static tests on Win32

7.47_11 Sun 20 Sep 10:02:15 BST 2020

    Enhancements:
    - Hint file loading returned to using eval() as documented

7.47_10 Tue 15 Sep 19:30:32 BST 2020

    Test fixes:
    - BareV2String should also be skipped on v5.8.0

7.47_09 Mon 14 Sep 14:33:52 BST 2020

    Bug fixes:
    - Removed duplicated NOOP on pure_all

    Test fixes:
    - Remove tests for invalid version declarations
    - Make windows specific code for NoXS compile under strict

7.47_08 Mon 31 Aug 09:43:01 BST 2020

    Test fixes:
    - Resolved race conditions so the whole testsuite
      can be run in parallel safely again

    QA Fixes:
    - Added github action for Strawberry Perl
    - Added github action for Ubuntu/Linux

7.47_07 Mon  3 Aug 22:34:08 BST 2020

    Bug fixes:
    - Ensure all modules have 'use warnings'
    - Resolve revealed problems with warnings

    Test fixes:
    - Ensure all tests have strict and warnings
    - Resolve revealed problems with strict and warnings

7.47_06 Sat  1 Aug 14:36:00 BST 2020

    Bug fixes:
    - Refine archname to use abi_version and abi_revision

7.47_05 Fri 31 Jul 10:35:54 BST 2020

    Bug fixes:
    - use $^V instead of PERL_VERSION after Perl 5.10

7.47_04 Tue 28 Jul 19:34:00 BST 2020

    Test fixes:
    - Skip static tests on AIX and z/OS when in
      core

    QA fixes:
    - Added v5.30 and v5.32 to travis build

7.47_03 Wed  8 Jul 21:58:29 BST 2020

    Test fixes:
    - Correct skip count in MM_Cygwin.t

7.47_02 Tue  7 Jul 01:40:13 BST 2020

    OS390 Enhancements:
    - Added MM subclass for OS390
    - Override xs_make_dynamic_lib() for os390

7.47_01 Fri 26 Jun 10:55:42 BST 2020

    Bug fixes:
    - Sanitise provided VERSION and VERSION_FROM
      ( RT#132875 )

7.46  Tue 23 Jun 10:19:40 BST 2020

    No changes since v7.45_01

7.45_01 Thu 28 May 17:29:45 BST 2020

    Bug fixes:
    - Always link to $Config{libs} on cygwin

    Test fixes:
    - Fix testing how some versions should be parsed

    QA fixes:
    - Add libnsl-devel to Cygwin packages we need

### CODE REVIEW

**Comment by toddr** [comment]
Do not merge yet. I'm investigating https://github.com/Perl-Toolchain-Gang/ExtUtils-MakeMaker/issues/372

**Comment by toddr** [comment]
> Do not merge yet. I'm investigating [Perl-Toolchain-Gang/ExtUtils-MakeMaker#372](https://github.com/Perl-Toolchain-Gang/ExtUtils-MakeMaker/issues/372)


This is fixed by a one line unit test change. I'm inclined to merge 7.48 and not worry about the 2 lines of noise in the test suite for the short term. Alternatively, I could add the change in blead, waiting on the next EUMM release.

https://github.com/Perl-Toolchain-Gang/ExtUtils-MakeMaker/pull/373

File: Porting/Maintainers.pl [hunk 1]
```diff
@@ -488,11 +488,6 @@ package Maintainers;
             'README.packaging',
             'lib/ExtUtils/MakeMaker/version/vpp.pm',
         ],
-        'CUSTOMIZED' => [
-            # https://github.com/Perl/perl5/issues/17601
-            # https://rt.cpan.org/Ticket/Display.html?id=115321
-            't/lib/MakeMaker/Test/Setup/XS.pm',
-        ],
```
Comment (tonycoz) [inline Porting/Maintainers.pl:490]: `t/porting/customized.dat` needs to be updated too

Comment (toddr) [reply Porting/Maintainers.pl:490]: Got it. Thanks!

**Pull request merged**